resources/user-agents/browsers/mozilla-browser/mozilla-browser-0-6-1-8.json

```diff
@@ -0,0 +1,85 @@
+{
+  "division": "Firefox #MAJORVER#.#MINORVER#",
+  "versions": ["1.8", "1.7", "1.6", "1.5", "1.4", "1.3", "1.2", "1.1", "1.0", "0.9", "0.8", "0.7", "0.6"],
+  "sortIndex": 3172,
+  "lite": false,
+  "standard": true,
+  "userAgents": [
+    {
+      "userAgent": "Mozilla #MAJORVER#.#MINORVER#",
+      "engine": "Gecko_Dynamic",
+      "device": "general Desktop",
+      "properties": {
+        "Parent": "DefaultProperties",
+        "Comment": "Mozilla #MAJORVER#.#MINORVER#",
+        "Browser": "Mozilla",
+        "Browser_Maker": "Mozilla Foundation",
+        "Browser_Type": "Browser",
+        "Version": "#MAJORVER#.#MINORVER#",
+        "MajorVer": "#MAJORVER#",
+        "MinorVer": "#MINORVER#",
+        "Frames": "true",
+        "IFrames": "true",
+        "Tables": "true",
+        "Cookies": "true",
+        "JavaScript": "true",
+        "CssVersion": "2"
+      },
+      "children": [
+        {
+          "match": "Mozilla/5.0 (#PLATFORM# rv:#MAJORVER#.#MINORVER#*) Gecko/200?????",
```

Comment:
The other thing I was careful about here was not using something like:

`Mozilla/5.0 (#PLATFORM# rv:#MAJORVER#.#MINORVER#*) Gecko/*` as a match here because that trailing asterisk will pick up a lot of patterns that should maybe be caught by a different pattern that is shorter than this one. Since this pattern is so generic, I wanted to make sure that didn't happen.

The reason I use `200?????` after `Gecko/` here is because no builds of Mozilla in the versions above should have happened outside of the 2000-2009 decade. So this limits the possible conflicts these patterns could have with other more modern patterns.  

I didn't add a trailing asterisk for the same reasons mentioned above. This does cause the pattern to miss some agents that have trailing information after the `Gecko/XXXXXXXX` bit, but I don't see a better way to deal with that without this pattern grabbing other agents that it shouldn't (other than lengthening a lot of other patterns).